packages/ember-css-modules/lib/modules-preprocessor.js

```diff
@@ -184,7 +191,7 @@ module.exports = class ModulesPreprocessor {
 
     return this._resolvePath(importPath, fromFile, {
       defaultExtension: this.owner.getFileExtension(),
-      ownerName: this.owner.getParentName(),
+      ownerName: this._parentName || this.owner.getParentName(),
```

Comment:
In what situation will `this._parentName` not be set? If not set, shouldn't it use the same logic as above where `_parentName` is set?

Reply:
I'm probably being overly cautious here and could just rely on `this._parentName` without a fallback.

Reply:
Yep, I removed `this._parentName` without build/runtime issues. 👍